### PR TITLE
add procedure and script to split up a review of our issue backlog

### PR DESCRIPTION
**Pull request by ca-scribner**
This is some simple instructions and a script to take out entire issue backlog and split it up, by repository, into groups of repositories with roughly the same number of issues.  This makes it easy for us to then each take a group of a few repos, review the open issues, and maintain our backlog.

See readme.md for more details

### CODE REVIEW

File: scripts/backlog-review/group_issues.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import typer
+import json
+
+
+def main(issue_file: str, n_groups: int):
+    """
+    Splits a list of {repo_name, n_open_issues} into n_groups, where each group has roughly the same number of issues.
```
Comment (ca-scribner) [inline scripts/backlog-review/group_issues.py:7]:
```suggestion
    Splits a list of repos into n_groups, where each group has roughly the same number of issues.
```

File: scripts/backlog-review/group_issues.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import typer
+import json
```
Comment (ca-scribner) [inline scripts/backlog-review/group_issues.py:2]:
```suggestion
import json

import typer
```

**Comment by misohu** [comment]
Haha love the script. Also did not know about typer before. (Maybe add some error handling in case sb sends wrong file).

**Comment by ca-scribner** [comment]
@misohu typer is fantastic.  Great when you want to do something lightweight

For error handling, I've added something more verbose.  what do you think?

**Review by misohu — APPROVED** [review]
Love it <3 great work

**Pull request merged**